eras/shelley/formal-spec/epoch.tex

```diff
@@ -571,13 +571,10 @@ \subsection{Pool Reaping Transition}
                  & \{\var{hk}\mapsto \fun{poolRAcnt}~\var{pool} \mid
                    \var{hk}\mapsto\var{pool} \in \var{retired}\restrictdom\var{poolParams} \} \\
         \var{rewardAcnts'} & \left\{
-                        a \mapsto c
-                        \mathrel{\Bigg|}
-                        \begin{array}{r@{~\in~}l}
-                          \var{hk} \mapsto c & \var{pr}, \\
-                          \var{hk}\mapsto\var{a} & \var{rewardAcnts} \\
-                        \end{array}
-                      \right\} \\
+                               \sum\limits_{\wcard\mapsto c\in\var{pr}\circ\var{rewardAcnts}^{-1}(a)} c
+                               \mathrel{\Bigg|}
+			       a\in\range{rewardAcnts}
+                             \right\} \\
```

Comment:
I think what you want is `a \mapsto \sum\limits_{c\in\var{pr}(\var{rewardAcnts}^{-1}(a))} c`:
- currently, `rewardAcnts'` is just a set of numbers
- `\var{rewardAcnts}^{-1}(a)` is a set of hashes, so you can only apply it to `pr`, not compose it with, which will give you a set of numbers

Also, I think the `|` looks a bit better if it's just `Big`. Finally, I think `\sum\var{pr}(\var{rewardAcnts}^{-1}(a))` is a nice point-free version of the sum, but pick whichever one you like better.

Reply:
you're totally right! I had meant for `rewardAcnts` to be a mapping. thanks!

Reply:
your point free version is very clean, I love it, I've changed to using it